<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="url_label">
        <property name="text">
         <string>url:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="url_line_edit"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_start_preview">
        <property name="text">
         <string>start_preview</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_stop_preview">
        <property name="text">
         <string>stop_preview</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_start_record">
        <property name="text">
         <string>start_record</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_stop_record">
        <property name="text">
         <string>stop_record</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_capture_frame">
        <property name="text">
         <string>capture_frame</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTabWidget" name="tab_widget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QPlayerWidget" name="player_widget">
       <attribute name="title">
        <string>player_widget</string>
       </attribute>
      </widget>
      <widget class="QGLPlayerWidget" name="gl_player_widget">
       <attribute name="title">
        <string>gl_player_widget</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGLPlayerWidget</class>
   <extends>QWidget</extends>
   <header location="global">QGLPlayerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QPlayerWidget</class>
   <extends>QWidget</extends>
   <header location="global">QPlayerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
